tasks/main.yml

```diff
@@ -52,121 +49,55 @@
   copy:
     src: "{{ item.src }}"
     dest: "{{ item.dest }}"
-  with_items: "{{ automation_extra_files }}"
-  when: automation_extra_files is defined
-
-#
-# Install Replicated
-#
-# Parameters are:
-#   - private_address
-#   - public_address
-#   - proxy
-#   - docker_version
-#   - install replicated|operator; replicated is the default
-#   - install_url overwrite the default install url
-#
-
-- name: Set install parameters
+  with_items: "{{ replicated_automation_extra_files }}"
+  when: replicated_automation_extra_files is defined
+
+###
+### Install Replicated
+###
+### Parameters are:
+###   - replicated_private_address
+###   - replicated_public_address
+###   - replicated_proxy
+###   - replicated_docker_version
```

Comment:
Feedback from Circle CI was to ensure that all variables are prefixed to avoid variable conflicts in different playbooks.